tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -35,15 +37,11 @@ std::vector<std::string> splitCommaSeparatedString(const std::string& comma_sepa
   std::vector<std::string> output;
   std::stringstream ss(comma_separated_string);
 
-      std::string string_element;
-      while (ss >> string_element) {
-          output.push_back(string_element);
-
-          if (ss.peek() == ',')
-              ss.ignore();
-      }
-
-   return output;
+  std::string string_element;
+  while (std::getline(ss, string_element, ',')) {
+    output.push_back(string_element);
+  }
+  return output;
```

Comment:
Sorry for making the whitespace error. FYI: Intermodalics style guide is 4 spaces :)